pages/common/sphinx-build.md

```diff
@@ -0,0 +1,16 @@
+# sphinx-build
+
+> Generates documentation from the files in sourcedir and places it in the outputdir.
+> More information: http://www.sphinx-doc.org/en/master/man/sphinx-build.html.
+
+- Build documentation as "epub" and put it in "builddir":
+
+`sphinx-build -b {{epub}} {{path/to/sourcedir}} {{path/to/builddir}}`
+
+- Build documentations intended for readthedocs.io:
+
+`pip install -U sphinx sphinx-rtd-theme ; sphinx-build -b {{html}} {{path/to/docs_dir}} {{path/to/builddir}}`
```

Comment:
```suggestion
`pip install -U sphinx sphinx-rtd-theme ; sphinx-build -b {{html}} {{path/to/docs_dir}} {{path/to/builddir}}`
```
I don't think there's the need to also show how to install the `sphinx-rtd-theme` package. It should be added in the description of the example, like this:

    - Build documentations intended for readthedocs.io (requires the sphinx-rtd-theme pip package):